### PR TITLE
Fix for the !create command and updation to the old discord bot libraries

### PR DESCRIPTION
**Pull request by tanmaypardeshi**
This PR has been created to achieve the following objectives:
* Fix the !create command for the TeachersPetBot
* Update the old discord libraries to the new ones.
* Updating the requirements file to the new libraries.

### CODE REVIEW

File: data/charts/chartstorage.json [hunk 1]
```diff
@@ -3,9 +3,12 @@
...
     },
     "grades": {
-        "URL": "https://tinyurl.com/y5u9btfn"
+        "URL": "https://tinyurl.com/yxnlq8tf"
     },
     "attendance": {
         "URL": "https://tinyurl.com/yxmv9sdj"
+    },
+    "class_average_between_6_semesters": {
+        "URL": "https://tinyurl.com/ys8o9p4p"
```
Comment (tanmaypardeshi) [inline data/charts/chartstorage.json:12]: When I open this URL, I see a static image. Just out of curiosity, I want to know how this works. Can you redirect me to some part of the code or documentation where I can understand this?

File: src/bot.py [hunk 2]
```diff
@@ -229,6 +229,7 @@ async def on_message(message):
     temp=[]
     ctx = await bot.get_context(message)
     print(message.content)
+    print(message.author.id)
```
Comment (tanmaypardeshi) [inline src/bot.py:232]: Are the print statements necessary in the main branch or are they just for debugging. They need to be removed if they are just for debugging.

File: test/test_qna.py [hunk 3]
```diff
@@ -29,13 +28,14 @@ async def test(testing_bot, guild_id):
...
     print('testing question')
     qna_channel = discord.utils.get(testing_bot.get_all_channels(), name='q-and-a')
-    await qna_channel.send('!ask \"Hello\"')
-
-    sleep(0.5)
-
-    messages = await qna_channel.history(limit=1).flatten()
+    await qna_channel.send('!ask Hello')
 
+    sleep(.5)
```
Comment (tanmaypardeshi) [inline test/test_qna.py:33]: What is exactly the purpose of using sleep if we are using async and await?

**Review by tanmaypardeshi — COMMENTED** [review]
I don't see any major issues and most of the code is very well written with comments!

File: src/bot.py (hunk 2 above)
Comment (ExtremeMachine12) [reply src/bot.py:232]: They are just for debugging

File: test/test_qna.py (hunk 3 above)
Comment (ExtremeMachine12) [reply test/test_qna.py:33]: To be honest, I'm not entirely sure, it's just the way the last group had it coded but I changed it a couple of times to see which is why it counted as a change on my end.

**Review by ExtremeMachine12 — APPROVED** [review]
Looks good

**Pull request merged**